impls/src/error.rs

```diff
@@ -101,6 +102,31 @@ impl Fail for Error {
 	}
 }
 
+// impl From<dyn error::Error + 'static> for Error {
+// 	fn from(error: libtx::Error) -> Error {
+// 		Error {
+// 			inner: Context::new(ErrorKind::LibTX(error.kind())),
+// 		}
+// 	}
+// }
+
+// impl From <dyn error::Error + 'static> for Error {
+// 	fn source(&self) -> Option<&(dyn Error + 'static)> {
+//         Some(&self.name)
+//     }
+// }
+// fn from(error: libwallet::Error) -> Error {
+// 	Error {
+// 		inner: Context::new(ErrorKind::LibWallet(error.kind(), format!("{}", error))),
+// 	}
+// }
+
+// impl error::Error for Error {
+// 	fn source(&self) -> Option<&(dyn error::Error + 'static)> {
+// 		Some(&self.inner)
+// 	}
+// }
+
```

Comment:
Are we confident enough about the current implementation that we can remove these commented lines?